src/main/java/org/jitsi/jigasi/SipGatewaySession.java

```diff
@@ -618,6 +618,15 @@ public void toneReceived(DTMFReceivedEvent dtmfReceivedEvent)
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean maybeUseAuthenticatedLogin()
+    {
+        return false;
```

Comment:
So this disables authenticated login for jigasi at all? Is this correct?

Reply:
Disabling it for dials using SIP

Reply:
We should not do that, people are already using it.
What you can do is add an account property that disables auth on xmpp account and act base on that. This way it will be per sip account in case of many sip accounts that serve multiple and different deployments.